capstone/heroku.md

```diff
@@ -10,42 +10,88 @@ $ brew tap heroku/brew && brew install heroku
 $ heroku --version
 ```
 
-3. Now we can login to Heroku from the command line. Enter your email and password for your group Heroku account.
+3. First we'll use Heroku's browser authenticated login in via the command line
+  - 3a. Login to the Heroku website with your groups Heroku credentials.  
+  - 3b. Run the following command 
 ```bash
-$ heroku login -i
+$ heroku login 
 ```
+  - 3c. When the new chrome page opens click the button that says "log in"
 
-4. Create a new app on Heroku. The url will be printed in the command line.
-```bash
-$ heroku create
+4. Once you are logged in find the button that says "create a new app" on the Heroku dashboard. 
+  - 4a. Name your app something similar to your capstone project. your app name will also be the name of your url
+  - 4b. The new app will have a a set of cli prompts. Run the prompts most similar to the following  
+```
+$ heroku git:remote -a <your-app-name>
+$ git commit -am "Making things better"
 ```
+  - 4c. Your project is now connected to heroku 
 
-5. Ensure that the remote git repo on Heroku was added to your local git config.
-```bash
-$ git config --list | grep heroku
+## Now we need to install some deployment tools and configurations
+
+5.  Run the following command to allow builds of your rails app to happen on intel chip based computers so the build can be sent to Heroku. 
+```
+$ bundle lock --add-platform x86_64-linux
+$ git add .
+$ git commit -m "added x86_64 linux to platforms" 
+```
+
+6. Update the bundle packages to support this platform adjustment
+```
+$ bundle update
+```
+
+7. If you have the master.key for the project in your config folder file skip this step.
+  - 7a. To create a new master.key and credential.yml file run the command
 ```
+$ EDITOR="code --wait" bin/rails credentials:edit
+```
+  - 7b. this will create a new master.key and it's associated credentials.yml.enc file
+
+```
+$ git add .
+$ git commit -m "updated credentials"
+```
+
 
-6. We need to add the `master.key` that your app uses to decrypt your `credentials.yml.enc` file.  Copy the value in `/config/master.key`, and use it in the following command:
+8. We need to add the `master.key` that your app uses to decrypt your `credentials.yml.enc` file. 
+ - 8a. Copy the value in `/config/master.key`, and use it in the following command:
 ```bash
 $ heroku config:set RAILS_MASTER_KEY=<your-master-key-here>
 ```
 
-7. Finally, we're ready to push our code.
+9. Now we need to configure the build packs that Heroku will utilize for creating the React on rails application
+``` 
+$ heroku buildpacks:add --index 1 heroku/nodejs
+$ heroku buildpacks:add --index 2 heroku/ruby
+```
+
+10. Finally, we're ready to push our code. Double check you are on the main branch of your project.
```

Comment:
Because some people may them the following terminal command is checking. Might then to state, `Next run the following command to push your local main branch to Heroku.`